recipe/activate-clang.sh

```diff
@@ -195,6 +201,25 @@ else
     diff -U 0 -rN /tmp/old-env-$$.txt /tmp/new-env-$$.txt | tail -n +4 | grep "^-.*\|^+.*" | grep -v "CONDA_BACKUP_" | sort
     rm -f /tmp/old-env-$$.txt /tmp/new-env-$$.txt || true
   fi
+
+  # fix prompt for zsh
+  if [[ -n "${ZSH_NAME}" ]]; then
+    _conda_clang_precmd() {
+      CONDA_CLANG_OLDHOST="${HOST}"
+      HOST="${CONDA_CLANG_HOSTNAME}"
```

Comment:
```suggestion
      HOST="${CONDA_BACKUP_HOST}"
```

Reply:
We can remove the other bits of the activater script this way. Good catch!